CMakeLists.txt

```diff
@@ -7,7 +7,7 @@
 cmake_minimum_required(VERSION 3.0.2)
 
 project("PELUX Baseline Software Factory"
-        VERSION 0.1.0
```

Comment:
Does the released Pelux 1.0 SWF state version 0.1.0?

Reply:
No, it states 1.0. That branch is 1 commit ahead of master in that sense.